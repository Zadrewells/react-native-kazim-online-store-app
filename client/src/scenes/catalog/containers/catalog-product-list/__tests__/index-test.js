import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import {
	CatalogProductList
} from 'src/scenes/catalog/containers';

describe('<Label />', () => {
	const getWrapper = props => renderer.create(
		<CatalogProductList
			{
			...props
			}
		/>
	).toJSON();

	const modelI = {
		title: 'teste'
	};

	it('renders correctly', () => {
		const tree = getWrapper(modelI);
		expect(tree).toMatchSnapshot();
	});
});
