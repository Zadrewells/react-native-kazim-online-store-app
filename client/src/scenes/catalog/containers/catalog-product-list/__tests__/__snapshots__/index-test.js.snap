// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> renders correctly 1`] = `
<View
  isFirstItem={false}
  isLastItem={false}
  style={
    Array [
      Object {
        "marginBottom": 32,
        "marginTop": 0,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "flex-start",
          "flexDirection": "row",
          "justifyContent": "flex-start",
          "marginBottom": 16,
          "marginLeft": 24,
          "marginRight": 0,
          "marginTop": 0,
        },
      ]
    }
  >
    <Text
      dark={true}
      fontSize={18}
      margin="0px 0px 5px 0px"
      style={
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontFamily": "quicksand-bold",
            "fontSize": 18,
            "marginBottom": 5,
            "marginLeft": 0,
            "marginRight": 0,
            "marginTop": 0,
            "textAlign": "center",
            "textAlignVertical": "center",
          },
        ]
      }
      text="teste"
    >
      teste
    </Text>
  </View>
</View>
`;
